examples/cancellablecontainers.jl

```diff
@@ -36,8 +36,8 @@ Base.eltype(::Type{<:Blocking{T}}) where {T} = T
 
 # The idea is to try sending or receiving the item via the channel and *"then"*
 # try to manipulate the data collection. We can do this *atomically* by using
-# the choice reagent [`|`](@ref Reagents.:|).  Note that the order of this
-# operation is importnat. For example, when invoking `putting(b)`, it tries
+# the [choice reagent `|`](@ref Reagents.:|). Note that the order of this
```

Comment:
```suggestion
# the choice reagent [`|`](@ref Reagents.:|). Note that the order of this
```

it looks like Documenter does not like this...

Hmm... I don't know why but

Reply:
Sure, no problem. Sorry I just assumed it would work!